lisa/analysis/base.py

```diff
@@ -131,7 +131,7 @@ def _save_plot(self, figure, default_dir, filepath=None, img_format=None, wrappe
             guessed_format = mime_type.split('/')[1].split('.', 1)[-1].split('+')[0]
             img_format = img_format or guessed_format
 
-        figure.savefig(filepath, format=img_format)
+        figure.savefig(filepath, format=img_format, bbox_inches='tight')
```

Comment:
Doesn't that cause issues with the resulting image file ? IIRC a handful of plot functions had this, but it caused more harm than good (usually stripped part of the title and such).

Reply:
On all the plots I tried to save so far, the title was always cropped without that option. I can make it the default only, so that plot functions needing something else can override it ?